packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -156,7 +156,14 @@ const ProductList = (props) => {
                 perPricebook: true,
                 allVariationProperties: true,
                 allImages: true,
-                expand: ['promotions', 'variations', 'prices', 'images', 'custom_properties'],
+                expand: [
+                    'promotions',
+                    'variations',
+                    'prices',
+                    'images',
+                    'custom_properties',
+                    'availability'
```

Comment:
We need this param to know the orderable status of each variant and filter them out the related swatch on product tile